h5p-bildetema-words-topic-image/language/nb.json

```
@@ -3,6 +3,9 @@
     {
       "label": "Bilde"
     },
+    {
+      "label": "Backend Url"
```

This can also be added, similar as .nb.json for h5p-bildetema

```suggestion
      "label": "Backend Url",
      "description": "The Url to the json database",
      "default": "https://cdn-devbildetema.azureedge.net/data/database.json",
```